File: TEAM B2 (Season 2)/Week 1/Moana/[완전탐색_모음사전].py
#완전탐색_모음사전
def solution(word):
    answer = 0
    vowels = ["A", "E", "I", "O", "U"]
    digits = [781, 156, 31, 6, 1]
    for i in range(len(word)):
        answer += vowels.index(word[i]) * digits[i] + 1
    return answer

File: TEAM B2 (Season 2)/Week 1/Musk/[완전탐색_모음사전].cpp
#include <string>
#include <vector>

using namespace std;

int cnt = -1;
int answer = 0;
string target = "";
string aeiou = "AEIOU";

void dfs(string word) {
     cnt++;
    
    if (word == target) {
        answer = cnt;
        return;
    }
    
    if (word.length() >= 5) return;
    
    
    for (int i=0; i<5; i++) {
        dfs(word + aeiou[i]);
    }
}

int solution(string word) {   
    target = word;
    dfs("");
    
    return answer;
}